test/gtest/ucp/test_ucp_am.cc

```diff
@@ -995,6 +995,14 @@ class test_ucp_am_nbx_eager_data_release : public test_ucp_am_nbx {
         ucp_am_data_release(receiver().worker(), m_data_ptr);
     }
 
+    size_t zcopy_fragment_size()
+    {
+        size_t max_zcopy = ucp_ep_config(sender().ep())->am.max_zcopy;
+        return max_zcopy > sizeof(ucp_am_hdr_t) ?
+                       max_zcopy - sizeof(ucp_am_hdr_t) :
+                       0;
```

Comment:
maybe just set RNDV_THRESH=inf and run the test on relatively large message size, such as 1mb?